### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by JackLivio**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,10 +64,15 @@ The Docker instance of Core exposes multiple ports for different types of commun
...
 | 12345          | SDL Core's port used to communicate with mobile application over TCP |
 
+
 ## Start the HMI
 
+Once SDL Core is setup, follow these steps to clone, build, and run the SDL Generic HMI.
+
 Clone this repository
 
+Note: If you are not making any changes to the generic hmi, you may skip straight to the last step and launch the Generic HMi in a web browser.
```
Comment (theresalech) [inline README.md]: Should update the first reference from "generic hmi" to "Generic HMI" and the second reference from "HMi" to "HMI"

File: README.md [hunk 2]
```diff
@@ -68,6 +116,14 @@ then go through the usage instructions again.
 
 The main third-party technologies we use to develop this HMI are React, React-Redux, and React-Router. Implement an SDL HMI is an exercise in receiving, processing, and responding to RPCs which are coming from a connected SDL Core instance.
 
+
+Note: After making any changes to the generic hmi, you must run
```
Comment (theresalech) [inline README.md]: "Generic HMI"

File: README.md [hunk 3]
```diff
@@ -68,6 +116,14 @@ then go through the usage instructions again.
 
 The main third-party technologies we use to develop this HMI are React, React-Redux, and React-Router. Implement an SDL HMI is an exercise in receiving, processing, and responding to RPCs which are coming from a connected SDL Core instance.
 
+
+Note: After making any changes to the generic hmi, you must run 
+```
+$ webpack
+```
+before relaunching the hmi in the browser to see any changes made.
```
Comment (theresalech) [inline README.md]: "HMI"

File: README.md [hunk 4]
```diff
@@ -2,6 +2,42 @@
...
+  1. [Clone the SDL Core repository](https://github.com/smartdevicelink/sdl_core)
+  2. Create a folder for your build and run `cmake ../sdl_core`
+  3. If there are any dependency issues, install missing dependencies:
+  
+  
+```
+sudo apt-get install git cmake build-essential libavahi-client-dev libsqlite3-dev chromium-browser libssl-dev libudev-dev libgtest-dev libbluetooth3 libbluetooth-dev bluez-tools gstreamer1.0* libpulse-dev
+```
+    
+  4. Run the following commands to compile and install smartdevicelink core
```
Comment (theresalech) [inline README.md]: "SDL Core"

**Comment by JackLivio** [comment]
@theresalech Thank you for your comments. I have made the requested changes.

File: README.md [hunk 5]
```diff
@@ -42,9 +83,16 @@ Install dependencies (you might need to clean the `node_modules` folder):
...
+
 ```
-$ webpack && npm start
+$ webpack
+```
+
+Launch the Generic HMI in a web browser
+
+```
+$ chromium index.html
```
Comment (jacobkeeler) [inline README.md]: `chromium` is not a command, should be `chromium-browser`

**Pull request merged**